### PR TITLE
Small visual fixes

### PR DESCRIPTION
**Pull request by patel1643**
- move submit buttons to user info section
- remove top padding for three buttons on sidenavbar

### CODE REVIEW

File: client/src/pages/Profile.tsx [hunk 1]
```diff
@@ -197,13 +197,23 @@ export const Profile = () => {
                                 </Box>
                             </Flex>
                             <div>
-                                <FormLabel>Username</FormLabel>
+                                <FormLabel>Display name</FormLabel>
```
Comment (Mwpereira) [inline client/src/pages/Profile.tsx:200]: Please capitalize the N

**Review by Mwpereira — CHANGES_REQUESTED** [review]
Please change **Username** to **Display Name** on the Register page 😀

File: client/src/pages/Profile.tsx (hunk 1 above)
Comment (patel1643) [reply client/src/pages/Profile.tsx:200]: wdym why would i capitalize "N" word

Comment (patel1643) [reply client/src/pages/Profile.tsx:200]: lol jk I will

**Comment by patel1643** [comment]
> Please change **Username** to **Display Name** on the Register page 😀

done sir

**Pull request merged**